packages/synth.queues/src/services/kafka.queue.service.ts

```diff
@@ -0,0 +1,118 @@
+import { PluginManager } from '@applicature/synth.plugin-manager';
+import * as kafka from 'node-rdkafka';
+import * as logger from 'winston';
+import { KafkaConfig, Message, Queue } from '../types';
+import { QueueService } from './queue.service';
+export class KafkaQueueService extends QueueService {
+    private producerReady: any;
```

Comment:
you may init producer in `init` method, and use an instance of producer, instead of Promise< Producer >

Reply:
While we're using Promise-instance of Producer/Consumer, we can use them instantly without adding another `.on`-methods. Is it bad?

Reply:
if you init them in `init` method, then you may store initialized Producer/KafkaConsumer instead of Promise<Producer/KafkaConsumer>, and after that, you will be able to use instances of Producer/KafkaConsumer in methods `receiveMessage`, and `sendMessage` without `this.consumerReady.then`, but use that instances directly, like `this.producer`, or `this.consumer`